develop/2017-04-29-fabiocorreacordeiro-Local-Transportation.py

```diff
@@ -0,0 +1,568 @@
+taxi_mean = taxi_mean.reset_index()
+taxi_mean = taxi_mean.groupby(['year',
+                               'month']).agg({'total_net_value':'mean'})
+
+#'Fuels and lubricants'
+fuel_mean = fuel.groupby(['year',
+                          'month',
+                          'congressperson_id']).agg({'total_net_value':sum})
+fuel_mean = fuel_mean.reset_index()
+fuel_mean = fuel_mean.groupby(['year',
+                               'month']).agg({'total_net_value':'mean'})
+
+#'Automotive vehicle renting or watercraft charter'
+rent_car_and_boat_mean = rent_car_and_boat.groupby(['year',
+                                                    'month',
+                                                    'congressperson_id']).agg({'total_net_value':sum})
+rent_car_and_boat_mean = rent_car_and_boat_mean.reset_index()
+rent_car_and_boat_mean = rent_car_and_boat_mean.groupby(['year',
+                                                         'month']).agg({'total_net_value':'mean'})
+
+#'Automotive vehicle renting or charter'
+rent_car_mean = rent_car.groupby(['year',
+                                  'month',
+                                  'congressperson_id']).agg({'total_net_value':sum})
+rent_car_mean = rent_car_mean.reset_index()
+rent_car_mean = rent_car_mean.groupby(['year',
+                                       'month']).agg({'total_net_value':'mean'})
+
+#'Watercraft renting or charter'
+rent_boat_mean = rent_boat.groupby(['year',
+
+                                    'month',
+                                    'congressperson_id']).agg({'total_net_value':sum})
+rent_boat_mean = rent_boat_mean.reset_index()
+rent_boat_mean = rent_boat_mean.groupby(['year',
+                                         'month']).agg({'total_net_value':'mean'})
+
+
+# In[100]:
+
+taxi_mean.plot()
+plt.title('Mean Expenses in "Taxi, toll and parking"')
+plt.ylim([0, 27000])
+
+fuel_mean.plot()
+plt.title('Mean Expenses in "Fuels and lubricants"')
+plt.ylim([0, 27000])
+
+rent_car_and_boat_mean.plot()
+plt.title('Mean Expenses in "Automotive vehicle renting or watercraft charter"')
+plt.ylim([0, 27000])
+
+rent_car_mean.plot()
+plt.title('Mean Expenses in "Automotive vehicle renting or charter"')
+plt.ylim([0, 27000])
+
+rent_boat_mean.plot()
+plt.title('Mean Expenses in "Watercraft renting or charter"')
+plt.ylim([0, 27000])
+
+
+# The "Taxi, toll and parking" expenses were much lower than the other subquotas, for now, we will not analyze it.
+
+# # "Fuels and lubricants"
+
+# When we read the legislation about CEAP (http://www2.camara.leg.br/legin/int/atomes/2009/atodamesa-43-21-maio-2009-588364-normaatualizada-cd-mesa.pdf) the Art 2°, § 1° said:
+# 
+# *"§ 1º As despesas estabelecidas nos incisos I, VII e VIII poderão ser realizadas por assessores, assim entendidos os servidores efetivos, os ocupantes de cargos de natureza especial ou secretários parlamentares vinculados à Câmara dos Deputados.*

```

Comment:
Here I think is important maintain the both languages because the law is in Portuguese, and interpretation/translation of this item is very important for this analysis.